plugins/ssr/ssrprinters.ml

```diff
@@ -74,12 +74,17 @@ let pr_term (k, c) = pr_guarded (guard_term k) pr_glob_constr_and_expr c
 let pr_hyp (SsrHyp (_, id)) = Id.print id
 let pr_hyps = pr_list pr_spc pr_hyp
 
+let pr_list_opt_surround l r spc pr ls =
```

Comment:
I don't find this combinator easy to understand, I think that the empty case should simply be expanded below as `if List.is_empty occ then mt () else ...`.

Reply:
Fixed.